### ISSUE TITLE
Paginating through facets modal does not work with Bootstrap 5

### ISSUE THREAD

**Issue by jcoyne** [projectblacklight/blacklight#2564]
Because this code is tightly coupled to Bootstrap 4: https://github.com/projectblacklight/blacklight/blob/main/app/javascript/blacklight/modal.js

`$(this).modal("hide")` is the BS4 API.

**Comment by jcoyne** [comment]
We might have to do something goofy like:
```
if (jQuery.fn.tooltip.Constructor.VERSION) {
 // Bootstrap 4 stuff
} else {
// Bootstrap 5
}
```

**Comment by jcoyne** [comment]
Should be fixed by #2575

**Comment by barmintor** [comment]
Closing as #2571 and #2575 should have addressed this, please re-open if necessary!
